docs/book/src/tasks/experimental-features/runtime-sdk/implement-topology-mutation-hook.md

```diff
@@ -134,15 +136,15 @@ kind: Cluster
 spec:
     topology:
       variables:
-        # namespace is not needed as this variable is global.
+        # `definitionFrom` is not needed as this variable does not have conflicting definitions.
         - name: no-proxy
           value: "internal.domain.com"
-        # namespaced variables require values for each individual schema.
+        # variables with the same name but different definitions require values for each individual schema.
         - name: http-proxy
-          namespace: inline
+          definitionFrom: inline
```

Comment:
Are there alternative names to this variable? Don't have better ideas right now, but not sure if it's immediately clear that this is where the _schema_ of the variable is defined

Reply:
I guess schemaFrom could be an alternative. We just used definitions in the ClusterClass.status so it's nice if that matches up. We used definitions there as it felt consistent to patches[].definitions.

But we could change both places to ClusterClass.status.variables[].schemas and Cluster.spec.topology.variables[].schemaFrom

Reply:
But I think I prefer definition / definitionFrom over schema / schemaFrom as we also have schema one level deeper already to describe the actual VariableSchema. So it's confusing if we have it on both levels.

Reply:
I'm +1 with definition, it is a consistent with what we have in clusterClass